.gitignore

```diff
@@ -1,3 +1,6 @@
+# beaker yaml
+guided-trout-2f805b9.yaml
```

Comment:
What is this?

Reply:
err... i didn't add that? it must be a gantry beaker yaml file based on the naming scheme.

Reply:
i'll remove in any case

Reply:
removed